sky/services/editing/editing.mojom

```diff
@@ -48,3 +48,13 @@ interface Keyboard {
   Show();
   Hide();
 };
+
+struct ClipData {
+  string text;
+};
+
+[ServiceName="editing::Clipboard"]
+interface Clipboard {
+  SetClipData(ClipData clip);
+  GetClipData() => (ClipData? clip);
```

Comment:
My vision was to return whatever was in the clipboard, and the client could query what type it got back. I don't have a strong preference for doing it one way or the other. Is there a reason the `format` argument is superior?